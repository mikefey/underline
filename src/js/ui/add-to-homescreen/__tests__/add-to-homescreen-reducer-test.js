import test from 'ava';
import {
  SHOW_ADD_TO_HOMESCREEN,
  HIDE_ADD_TO_HOMESCREEN,
} from './../add-to-homescreen-constants';
import addToHomescreenReducer from './../add-to-homescreen-reducer.js';

test('default state', (t) => {
  t.snapshot(addToHomescreenReducer(undefined, {}));
});

test('handles SHOW_ADD_TO_HOMESCREEN', (t) => {
  t.snapshot(addToHomescreenReducer(undefined, {
    type: SHOW_ADD_TO_HOMESCREEN,
    os: 'Android',
  }));
});

test('handles HIDE_ADD_TO_HOMESCREEN', (t) => {
  t.snapshot(addToHomescreenReducer(undefined, { type: HIDE_ADD_TO_HOMESCREEN }));
});
