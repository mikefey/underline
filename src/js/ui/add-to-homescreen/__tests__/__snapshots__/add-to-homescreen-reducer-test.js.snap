exports[`default state 1`] = `"{\"show\":false,\"os\":null}"`;

exports[`handles HIDE_ADD_TO_HOMESCREEN 1`] = `"{"show":false,"os":null}"`;

exports[`handles SHOW_ADD_TO_HOMESCREEN 1`] = `"{\"show\":true,\"os\":\"Android\"}"`;
